doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -207,7 +207,7 @@ Machine learning
 Asset Tracker v2
 ----------------
 
-|no_changes_yet_note|
+* Updated the Wi-Fi configurations to optimize the RAM usage by about 25 kB for nRF91 Series DK and 12kB for Thingy:91 X.
```

Comment:
```suggestion
* Updated the Wi-Fi configurations to decrease the RAM usage by about 25 kB for the nRF91 Series DK and 12kB for the Thingy:91 X.
```

Reply:
Maybe optimize -> reduce in all of these. I believe the RAM usage decreases as the result of optimization.

Reply:
I'm using "reduce". Sounds better for me than "decrease" but I have no idea why. Let me know if it should be "decrease".